snapcraft/extractors/_metadata.py

```diff
@@ -48,10 +47,7 @@ def __init__(
         :param list desktop_file_paths: Extracted desktop file paths
         """  # noqa
 
-        if _data:
-            self._data = _data
-        else:
-            self._data = {}
+        self._data = {}  # type: Dict[str, Union[str, List[str]]]
```

Comment:
this is just me being blind, but I really started to prefer using `dict()` for these cases